nirc_ehr/resources/queries/study/housing.js

```diff
@@ -12,21 +33,16 @@ EHR.Server.TriggerManager.registerHandlerForQuery(EHR.Server.TriggerManager.Even
             // [2] =  To Location,
             // [3] = 28 - F2/CW - Rm 205 - B03
             // translate [3] to locationId
-            LABKEY.Query.selectRows({
-                schemaName: 'nirc_ehr',
-                queryName: 'locations',
-                columns: ['locationId'],
-                filterArray: [
-                    LABKEY.Filter.create('Name', locationTransferText[3].trim())
-                ],
-                scope: this,
-                success: function (data) {
-                    if (data && data.rows.length > 0) {
-                        let rec = data.rows[0];
-                        row.location = rec.LocationId;
-                    }
-                }
-            });
+            var location;
+            if (locationsData) {
+                location = locationsData[locationTransferText[3].trim()];
```

Comment:
Do we know that there will always be at least 4 elements in this array?

Reply:
I found location id in animal event table so parsing not needed and deleted this file.